bin/run-in-docker.sh

```diff
@@ -2,41 +2,42 @@
 set -e
 
 # Synopsis:
-# Test runner for run.sh in a docker container
-# Takes the same arguments as run.sh (EXCEPT THAT SOLUTION AND OUTPUT PATH ARE RELATIVE)
-# Builds the Dockerfile
-# Runs the docker image passing along the initial arguments
+# Run the test runner on a solution using the test runner Docker image.
+# The test runner Docker image is built automatically.
 
 # Arguments:
 # $1: exercise slug
-# $2: **RELATIVE** path to solution folder (with trailing slash)
-# $3: **RELATIVE** path to output directory (with trailing slash)
+# $2: absolute path to solution folder
```

Comment:
The trailing slash part is probably still important. `absolute` only means it starts with a drive, mount or `/`